package.json

```diff
@@ -8,6 +8,7 @@
     "dev": "webpack -w -c ./.config/webpack/webpack.config.ts --env development",
     "test": "jest --watch --onlyChanged",
     "test:ci": "jest --passWithNoTests --maxWorkers 4",
+    "test:template": "npx plop datasource Test && npx tsc --noEmit && rimraf src/datasources/test",
```

Comment:
I added this script to make it easy to test changes to the data source template. It generates a new data source, sees if it compiles, and then deletes it.